src/dss/main.py

```diff
@@ -30,5 +31,40 @@ def initialize_command(kubeconfig: str) -> None:
     initialize(lightkube_client=lightkube_client)
 
 
+CREATE_NOTEBOOK_IMAGE_HELP = """
+\b\nThe image used for the notebook server.  Suggested images:
+- kubeflownotebookswg/jupyter-scipy:v1.8.0
+- kubeflownotebookswg/jupyter-pytorch-full:v1.8.0
+- kubeflownotebookswg/jupyter-pytorch-cuda-full:v1.8.0
+- kubeflownotebookswg/jupyter-tensorflow-full:v1.8.0
+- kubeflownotebookswg/jupyter-tensorflow-cuda-full:v1.8.0
+"""
+
+
+@main.command(name="create-notebook")
+@click.option(
+    "--name",
+    help="Name given to the notebook being created.",
+)
+@click.option(
```

Comment:
?

```
@main.command(name="create-notebook")
@click.option(
```

Reply:
I'm not sure what you're asking, can you elaborate?

Reply:
Oh, sorry it was not clear. Aren't we missing the `@main.command(name="create-notebook")` decorator?

Reply:
oh I think that's [up here](https://github.com/canonical/data-science-stack/pull/43/files/627aff24391098bdb7a0506f3b11f173735eae7c#diff-2f57dc654b74f69ba83e508cc30ef64d0c75f5d95664138c59e693663019d467R44)